Product/SoapUI_Test/RegressionSuite/Passthru/AuditLogging-Passthrough/COREX12BatchSubmitRequest.properties

```diff
@@ -52,7 +52,7 @@ ActiveParticipant_Dest.RoleIDCode.@codeSystemName=DCM
 ActiveParticipant_Dest.RoleIDCode.@displayName=Destination
 
 AuditSourceIdentification.@AuditEnterpriseSiteID=Gateway 1
-AuditSourceIdentification.@AuditSourceID=1.1
+AuditSourceIdentification.@AuditSourceID=urn:oid:1.1
```

Comment:
Are we using this value as part of the request, or as part of an assertion later?